include/variant_detection/bam_functions.hpp

```diff
@@ -5,6 +5,26 @@
 #include <seqan3/io/sam_file/output.hpp>
 #include <seqan3/utility/char_operations/predicate.hpp>
 
+
+/*!\details
+ * From the SAM/BAM Format Specification
+ * | Bit  | Bit   | Description                                                        | Note                                       |
+ * |:----:|:-----:|:------------------------------------------------------------------:|:------------------------------------------:|
+ * |    1 | 0x1   | template having multiple segments in sequencing                    | necessary for read pair method             |
+ * |    2 | 0x2   | each segment properly aligned according to the aligner             | not checked yet - aligned as expected, probably not containing SVs |
```

Comment:
Does the `not checked yet` mean that this flag is not yet evaluated by iGenVar? If yes, `0x20`, `0x40`,`0x80` and `0x200` are also not checked yet. To save some space you could add a star * to every flag that is not evaluated yet and explain the star below the table?